keyboards/ergodone/keymaps/vega/keymap.c

```diff
@@ -0,0 +1,606 @@
+	KC_PGDN,	X(subsetof),	X(intersection),	X(angl),	X(nmember),	X(eleof),	KC_RSPC,
+	KC_RCTL, KC_RALT, KC_APP, TO(0), TO(0),
+	KC_END, KC_RBRC, KC_LEFT, KC_RGHT, KC_ENT, KC_SPC
+),
+
+[SYMB] = LAYOUT_ergodox(
+	X(Os), X(Oa), X(Ob), X(Oc), X(Od), X(Oe), X(mdot),
+	X(boxemp), X(bbstr), X(bbrtr), X(bbrtl), X(bbstl), X(degree), X(brkdn),
+	X(boxchk), X(bbmbl), X(bbml), X(bbmr), X(bbmbr), X(neteen),
+	X(boxX), X(bbsbr), X(bbrbr), X(bbrbl), X(bbsbl), X(uxclm), X(brkup),
+	X(floppy), TO(0), TO(0), X(arwu), X(arwd),
+	X(fire), X(lshade), X(mshade), KC_SPC, X(OS), X(dshade),
+
+	X(Ox), X(Of), X(Og), X(Oh), X(Oi), X(OA), X(OB),
+	X(numero), X(trade), X(copy), X(cleft), X(cent), X(OED), X(OC),
+	X(Agrave), X(gnd), X(sqr), X(sine), X(opt), X(OD),
+	X(sect), X(Aacute), X(Acircm), X(Adiaer), X(Abreve), X(Atilde), X(OE),
+	X(arwl), X(arwr), X(geq), X(leq), X(OF),
+	X(rang), X(water), X(perup), X(perdn), X(baster), KC_ENT
+),
+
+[GREL] = LAYOUT_ergodox(
+	KC_ESC, X(Rone), X(Rtwo), X(Rthree), X(Rfour), X(Rfive), KC_GRV,
+	KC_TAB, KC_QUOT, KC_COMM, KC_DOT, X(gp), X(gy), KC_SLSH,
+	KC_SLSH, X(ga), X(go), X(ge), X(gu), X(gi),
+	MO(GREU), KC_SCLN, X(gq), X(gj), X(gk), X(gx), KC_AMPR,
+	KC_MS_L, TO(0), TO(0), KC_INS, KC_DEL,
+	KC_LBRC, KC_HOME, KC_UP, KC_SPC, KC_LGUI, KC_DOWN,
+
+	TO(0), X(Rsix), X(Rseven), X(Reight), X(Rnine), X(Rten), KC_BSPC,
+	KC_PGUP, X(gf), X(gg), X(gc), X(gr), X(gl), KC_EQL,
+	X(gd), X(gh), X(gt), X(gn), X(gs), KC_MINS,
+	KC_PGDN, X(gb), X(gm), X(gw), X(gv), X(gz), MO(GREU),
+	KC_RCTL, KC_RALT, KC_APP, TO(0), TO(0),
+	KC_END, KC_RBRC, KC_LEFT, KC_RGHT, KC_ENT, KC_SPC
+),
+
+[GREU] = LAYOUT_ergodox(
+	KC_ESC, X(Rone), X(Rtwo), X(Rthree), X(Rfour), X(Rfive), KC_GRV,
+	KC_TAB, KC_QUOT, KC_COMM, KC_DOT, X(Gp), X(Gy), KC_SLSH,
+	KC_SLSH, X(Ga), X(Go), X(Ge), X(Gu), X(Gi),
+	KC_TRNS, KC_SCLN, X(Gq), X(Gj), X(Gk), X(Gx), KC_AMPR,
+	KC_MS_L, TO(0), TO(0), KC_INS, KC_DEL,
+	KC_LBRC, KC_HOME, KC_UP, KC_SPC, KC_LGUI, KC_DOWN,
+
+	TO(0), X(Rsix), X(Rseven), X(Reight), X(Rnine), X(Rten), KC_BSPC,
+	KC_PGUP, X(Gf), X(Gg), X(Gc), X(Gr), X(Gl), KC_EQL,
+	X(Gd), X(Gh), X(Gt), X(Gn), X(Gs), KC_MINS,
+	KC_PGDN, X(Gb), X(Gm), X(Gw), X(Gv), X(Gz), KC_TRNS,
+	KC_RCTL, KC_RALT, KC_APP, TO(0), TO(0),
+	KC_END, KC_RBRC, KC_LEFT, KC_RGHT, KC_ENT, KC_SPC
+),
+
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
+{
+  // MACRODOWN only works in this function
+      switch(id) {
+        case 0:
+        if (record->event.pressed) {
+          SEND_STRING (QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
+        }
+        break;
+        case 1:
+        if (record->event.pressed) { // For resetting EEPROM
+          eeconfig_init();
+        }
+        break;
+      }
+    return MACRO_NONE;
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    // dynamically generate these.
+    case EPRM:
+      if (record->event.pressed) {
+        eeconfig_init();
+      }
+      return false;
```

Comment:
`break` after `return` is dead code.

If you want a keycode handled here to continue being processed by QMK (for example for One-Shot stuff), you should either `break` or `return true` in each `case`. Otherwise, just the `return false` will do.